conf/application.conf

```diff
@@ -27,6 +27,12 @@ play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.FrontendFilters"
 
 play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
 
+play.filters.enabled += "uk.gov.hmrc.play.bootstrap.frontend.filters.AllowlistFilter"
```

Comment:
so we actually dont want to add this one in as we want the allow list filter disabled locally so we need to remove this from app config and we will then override this in prod. Sorry I should have made the more clear in our VERY brief catch up this morning 😀